scripts/mkcloud

```diff
@@ -210,6 +212,8 @@ function sshrun()
         export nosetestparameters=${nosetestparameters} ;
         export tempestoptions='${tempestoptions}' ;
         export cephvolumenumber=$cephvolumenumber ;
+        export raidvolumenumber=$raidvolumenumber ;
+        export raidtype=$raidtype ;
```

Comment:
maybe rename raidtype variable to want_raidtype - then it gets auto-forwarded into qa_crowbarsetup. and document it in onadmin_help function and move the default value into qa_crowbarsetup as well, because it has nothing to do with mkcloud script. This allows us to use it for physical clouds.